persistence/src/main/java/cz/muni/fi/pa165/mushrooms/entity/MushroomHunter.java

```diff
@@ -24,7 +24,7 @@
     @NotNull

```

Comment:
MushroomHunter should not be modified in this pull request. Particularly because this causes conflicts with project master branch.